Document/0x05h-Testing-Platform-Interaction.md

```diff
@@ -876,23 +1074,14 @@ In this case we've crafted the deep link including arbitrary parameters (`?messa
 - File: `WebViewActivity.kt`
 - Class: `com.mstg.deeplinkdemo.WebViewActivity`
 - Method: `onCreate`
-  
+
 > Sometimes you can even take advantage of other applications that you know interact with your target app. You can reverse engineer the app, (e.g. to extract all strings and filter those which include the target deep links, `deeplinkdemo:///load.html` in the previous case), or use them as triggers, while hooking the app as previously discussed.
 
 ## Testing for Sensitive Functionality Exposure Through IPC (MSTG-PLATFORM-4)
 
 ### Overview
 
-During implementation of a mobile application, developers may apply traditional techniques for IPC (such as using shared files or network sockets). The IPC system functionality offered by mobile application platforms should be used because it is much more mature than traditional techniques. Using IPC mechanisms with no security in mind may cause the application to leak or expose sensitive data.
-
-The following is a list of Android IPC Mechanisms that may expose sensitive data:
-
-- [Binders](https://developer.android.com/reference/android/os/Binder.html "IPCBinder")
-- [Services](https://developer.android.com/guide/components/services.html "IPCServices")
-- [Bound Services](https://developer.android.com/guide/components/bound-services.html "BoundServices")
-- [AIDL](https://developer.android.com/guide/components/aidl.html "AIDL")
-- [Intents](https://developer.android.com/reference/android/content/Intent.html "IPCIntent")
-- [Content Providers](https://developer.android.com/reference/android/content/ContentProvider.html "IPCContentProviders")
+To test for [sensitive functionality exposure through IPC](#sensitive-functionality-exposure-through-ipc "Sensitive Functionality Exposure Through IPC") mechanisms we will identify a list of IPC mechanisms our app uses, then review the source code to see whether sensitive data is leaked when the mechanisms are used.
 
 ### Static Analysis
```

Comment:
I can't send a suggestion for this. Please change 

> We start by looking at the AndroidManifest.xml, where all activities, services, and content providers included in the source code must be declared (otherwise the system won't recognize them and they won't run).

to

> We start by looking at the AndroidManifest.xml, where all activities, services, and content providers included in the app must be declared (otherwise the system won't recognize them and they won't run).